### DISCUSSION TITLE
utoipa-2.4.2

### DISCUSSION THREAD

**Discussion by juhaku** [juhaku/utoipa#367] [Announcements]
## What's New :gem: :new: :tada:
* (811a4c4) Update next versions
* (f35bcc4) Fix `ToSchema` map field exmaple (#365) @juhaku 
* (80c449a) Update Swagger UI release version
* (c362b52) Update Swagger UI version
* (98c2929) Use OUT_DIR in build.rs (#344) @dkellner 

#### Full [change log](https://github.com/juhaku/utoipa/compare/utoipa-2.4.1...utoipa-2.4.2)

<hr /><em>This discussion was created from the release <a href='https://github.com/juhaku/utoipa/releases/tag/utoipa-2.4.2'>utoipa-2.4.2</a>.</em>

**Comment by azzamsa** [comment]
I think we need a new release after axum dependency upgraded to v6.0

**Reply by juhaku** [reply]
New utoipa-swagger-ui contains the latest axum release.
